examples/http-proxy/README.md

```diff
@@ -46,7 +43,7 @@ libp2p-peer addresses:
 The run the local peer, indicating that it will need to forward http requests to the remote peer as follows:
 
 ```
-$ ./http-proxy -d /ip4/127.0.0.1/tcp/12000/ipfs/QmddTrQXhA9AkCpXPTkcY7e22NK73TwkUms3a44DhTKJTD
+> ./http-proxy -d /ip4/127.0.0.1/tcp/12000/ipfs/QmddTrQXhA9AkCpXPTkcY7e22NK73TwkUms3a44DhTKJTD
```

Comment:
Any particular reason for this change? Historically, `$` has meant "prompt where user is not root" and `#` has meant "prompt where user is root".